src/app/components/sites/pages/delete/delete.component.ts

```diff
@@ -84,6 +84,7 @@ SiteDeleteComponent.linkToRoute({
     [projectKey]: projectResolvers.show,
     [siteKey]: siteResolvers.show,
   },
+  title: () => "Delete",
```

Comment:
add to list of common titles

Reply:
With the addition of delete modals in #2042, there will be no need for the route title anymore